### PR TITLE
chore: position manager vault test case

### PR DESCRIPTION
**Pull request by Chef-Yogi**
### Summary
🔢🧪🗺️

Added tests and constants for position manager vaults. Reordered imports in `tokens.test.ts`.

> _`vaultsConfigChainMap`_
> _Exports vaults for each chain_
> _A useful constant_

### Walkthrough
* Export vaults configuration for each chain id in `vaultsConfigChainMap` constant ([link](https://github.com/pancakeswap/pancake-frontend/pull/8447/files?diff=unified&w=0#diff-726aec993475925573bef915f00996837628dc070b5565decbdd6b20eda72cbdR1-R8))
* Reorder import statements alphabetically in `tokens.test.ts` ([link](https://github.com/pancakeswap/pancake-frontend/pull/8447/files?diff=unified&w=0#diff-fef75ac495586128625e8a7b6f36f5820a6475ae0c99d23e30e72e2f60f5bf31L3-R7))
* Add test file `vaults.test.ts` to check the validity of the vaults configuration on different chains ([link](https://github.com/pancakeswap/pancake-frontend/pull/8447/files?diff=unified&w=0#diff-a4873ef8c9b426fec67f851d2715ff5e71540ce6cc56fdb9578c0ff36966645dR1-R49))

### CODE REVIEW

**Comment by chefjackson** [comment]
did we just found a wrong config from the unit test?

File: apps/web/src/config/__tests__/vaults.test.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import { ChainId } from '@pancakeswap/chains'
+import { vaultsConfigChainMap } from '@pancakeswap/position-managers/constants'
+import { positionManagerAdapterABI, positionManagerWrapperABI } from '@pancakeswap/position-managers/src/abi'
+import { pancakeV3PoolABI } from '@pancakeswap/smart-router/evm/abis/IPancakeV3Pool'
```
Comment (chefjackson) [inline apps/web/src/config/__tests__/vaults.test.ts:4]:
```suggestion
import { pancakeV3PoolABI } from '@pancakeswap/smart-router'
```

Comment (chefjackson) [reply apps/web/src/config/__tests__/vaults.test.ts:4]: oops sorry I don't think the abi is exported from the package

Comment (chefjackson) [reply apps/web/src/config/__tests__/vaults.test.ts:4]: although I think it should be exported by v3-sdk

**Comment by Chef-Yogi** [comment]
@chefjackson Do you think I should add a new one under apps/web ?

**Comment by chefjackson** [comment]
> @chefjackson Do you think I should add a new one under apps/web ?

What about we move the v3 pool abi into `v3-sdk` and import from there?

**Comment by Chef-Yogi** [comment]
sure, no problem

**Comment by Chef-Yogi** [comment]
will update it

File: apps/web/src/config/__tests__/vaults.test.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import { describe, expect, it } from 'vitest'
+
+const mainnetVaults = [vaultsConfigChainMap[ChainId.BSC], vaultsConfigChainMap[ChainId.ETHEREUM]]
+
+function hasDuplicates(array: any[]) {
+  return new Set(array).size !== array.length
+}
+
+describe('Config position manger Vault', () => {
+  it.each(mainnetVaults)('All farm has an unique pid', (...vaults) => {
```
Comment (ChefMomota) [inline apps/web/src/config/__tests__/vaults.test.ts:15]: Should be unique `id` right?  And this is `position manager` not `farm`

Comment (Chef-Yogi) [reply apps/web/src/config/__tests__/vaults.test.ts:15]: yes, let me update it

File: apps/web/src/config/__tests__/vaults.test.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
+import { ChainId } from '@pancakeswap/chains'
+import { vaultsConfigChainMap } from '@pancakeswap/position-managers/constants'
+import { positionManagerAdapterABI, positionManagerWrapperABI } from '@pancakeswap/position-managers/src/abi'
+import { pancakeV3PoolABI } from '@pancakeswap/v3-sdk'
+import { publicClient } from 'utils/client'
+import { describe, expect, it } from 'vitest'
+
+const mainnetVaults = [vaultsConfigChainMap[ChainId.BSC], vaultsConfigChainMap[ChainId.ETHEREUM]]
+
+function hasDuplicates(array: any[]) {
```
Comment (ChefMomota) [inline apps/web/src/config/__tests__/vaults.test.ts:10]: Type should be `Array<number>` ?

**Review by chefjackson — APPROVED** [review]

**Pull request merged**